creator-node/src/services/stateMachineManager/index.js

```diff
@@ -72,6 +102,62 @@ class StateMachineManager {
     this.highestEnabledReconfigMode = highestEnabledReconfigMode
     this.enabledReconfigModesSet = enabledReconfigModesSet
   }
+
+  _makeEnqueueJobsOnCompletion(monitoringQueue, reconciliationQueue) {
```

Comment:
slick, this is a very important function - should def document + think about making this very easy to understand

Reply:
actually per my other comment in Utils, this could realistically be its own file with long explanatory comment